spynnaker/pyNN/models/spike_source/spike_source_poisson_machine_vertex.py

```diff
@@ -81,12 +105,34 @@ def get_sdram_edge_params_bytes(vertex_slice):
         vertex_slice.n_atoms * SDRAM_EDGE_PARAMS_BYTES_PER_WEIGHT)
 
 
+def _u3232_to_uint64(array):
+    """ Convert data to be written in U3232 to uint32 array
```

Comment:
uint32 in description should be uint64 I think

Reply:
Oh yes, fixed